gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -7,17 +7,32 @@
 module PostgresHaAdmin
   class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
-    DB_CONNECTED_CHECK_FREQUENCY = 300
+    DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    attr_accessor :failover_attempts, :db_check_frequency, :failover_check_frequency
 
     def initialize(db_yml_file = '/var/www/miq/vmdb/config/database.yml',
                    failover_yml_file = '/var/www/miq/vmdb/config/failover_databases.yml',
+                   ha_admin_yml_file = '/var/www/miq/vmdb/config/ha_admin.yml',
                    log_file = '/var/www/miq/vmdb/log/ha_admin.log',
                    environment = 'production')
       @logger = Logger.new(log_file)
```

Comment:
a Logger should be optional, and should just take a logger object directly...  This will allow us to pass a VMDBLogger or MirroredLogger in ManageIQ and use a NullLogger or regular Logger in specs and in a separate gem.